backend.py

```diff
@@ -321,6 +355,66 @@ def getLinkInfo(self):
 
         return info
 
+    def getLootLog(self, loot_file: Union[str, List[str]], reversed: bool = True) -> List[LootItemData]:
+        """
+        Fetches the loot log from exported RClootcouncil file
+        :param loot_file:
+            Filename, list of filenames or directory containing .json files that contain the loot data
+        :param reversed:
+            If reversed, loot is sorted in time-ascending order
+        :return:
+            List containing the rows of BBCode for the equipment log
+        """
+        # Pull data from file(s)
+        pulled_data = []
+        indexes = []
+        if isinstance(loot_file, list):
+            for filename in loot_file:
+                with open(filename) as f:
+                    reader = csv.DictReader(f)
+                    for row in reader:
+                        pulled_data.append(dict(row))
+        elif os.path.isdir(loot_file):
+            files = [os.path.join(loot_file, f) for f in os.listdir(loot_file) if
+                     os.path.isfile(os.path.join(loot_file, f)) and '.json' in f]
```

Comment:
Do we actually support .json files?

Reply:
No, we just forget to change the file extension at 1:30 in the morning. Fixed.